atom/common/api/remote_callback_freer.cc

```diff
@@ -34,8 +34,7 @@ RemoteCallbackFreer::RemoteCallbackFreer(v8::Isolate* isolate,
 RemoteCallbackFreer::~RemoteCallbackFreer() {}
 
 void RemoteCallbackFreer::RunDestructor() {
-  base::string16 channel =
-      base::ASCIIToUTF16("ELECTRON_RENDERER_RELEASE_CALLBACK");
+  auto channel = "ELECTRON_RENDERER_RELEASE_CALLBACK";
```

Comment:
FTBFS:

```text
../../electron/atom/common/api/remote_callback_freer.cc:37:3: error: [chromium-style] auto variable type must not deduce to a raw pointer type.
  auto channel = "ELECTRON_RENDERER_RELEASE_CALLBACK";
  ^~~~
  auto*
1 error generated.
[17094/17960] CXX obj/electron/electron_lib/atom_renderer_client.o
[17095/17960] CXX obj/electron/electron_lib/atom_render_frame_observer.o
```

https://groups.google.com/a/chromium.org/forum/#!topic/chromium-dev/PMxDx7eR_YY has this info:

> the solution is typically to make the pointer part of the type be outside of auto, so instead of
>
> auto foo = new Foo;
>
> write
>
> auto* foo = new Foo;

Reply:
@ckerr fixed